sprocs/assessment_instances_grade.sql

```diff
@@ -17,20 +17,14 @@ DECLARE
     log_update boolean;
     use_credit integer;
     points DOUBLE PRECISION;
-    points_in_grading DOUBLE PRECISION;
```

Comment:
It looks as though `points_in_grading` and `score_perc_in_grading` are no longer used anywhere, is that correct? If so, can you open a ticket to remove all associated columns once this PR has been deployed?

Reply:
Done in #5911